pkg/strace/tracer.go

```diff
@@ -368,6 +379,12 @@ func (t *tracer) runLoop() error {
 			// Either a regular signal-delivery-stop, or a PTRACE_EVENT stop.
 			case syscall.SIGTRAP:
 				switch tc := status.TrapCause(); tc {
+				case unix.PTRACE_EVENT_SECCOMP:
+					// Handle seccomp event by continuing the syscall.
+					if err := syscall.PtraceSyscall(pid, 0); err != nil {
+						fmt.Println("errrrrror")
```

Comment:
ah, sorry I missed this, but printing like this in a package is a strict Nono :-)

Isn't there some way to avoid this? Any ideas? 
Sorry.

Reply:
Hahaha, sorry I forgot to replace it (I was using it for testing)